shell.nix

```diff
@@ -11,6 +11,7 @@ in
         zlib 
         openssl 
         cargo 
+        clippy
```

Comment:
Added clippy, can be run with "cargo clippy" and shows possible code improvements

Reply:
Ok, let's try it out.